server/src/tests/testDbSetup.js

```diff
@@ -0,0 +1,16 @@
+import mongoose from 'mongoose';
+
+const TEST_CONNECTION_STRING = 'mongodb://127.0.0.1:27017/test-db';
```

Comment:
Are you sure this database will work and CI server and all tests will pass?

Reply:
@rakeshtembhurne, Yes, the CI tests are working.